### DISCUSSION TITLE
Issue with filter displaying multi-select combo box

### DISCUSSION THREAD

**Discussion by shawkatmarrouf** [YetiForceCompany/YetiForceCRM#17497] [How to]
After upgrading to yetiforce 7, there is an issue with filter displaying values of a multi-select combo box field, I've tried it on the demo, and it gave me the same issue, it do like this (screen shot taken from YT online demo):
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/101041720/d26098c8-7d01-469c-8b60-b3a0f24b8593)

as you can see that values are going through the other columns in filter, and this so annoying, Is there any solution along with this issue also related to the filter https://github.com/YetiForceCompany/YetiForceCRM/issues/17482

**Comment by rskrzypczak** [comment]
Thanks, fix in YF7.0.2
